R/adverb-safely.R

```diff
@@ -10,7 +10,11 @@
 #' function (a verb). If you'd like to include a function created an adverb
 #' in a package, be sure to read [faq-adverbs-export].
 #'
-#' @inheritParams map
+#' @param .f A function to modify, specified in one of the following ways:
+#'   * A named function, e.g. `mean`.
+#'   * An anonymous function, e.g. `function(x) x + 1`.
+#'   * A lambda function, e.g. `\(a) a + 1`.
```

Comment:
Could we say that `function` is an anonymous function, and `\` is the shorthand for anonymous functions? Or call them both lambdas, but I'm not sure the term "lambda" brings any value.

In any case, making a distinction between anonymous functions and lambda functions might wrongly suggest they are something different rather than two ways of writing the same thing.